### PR TITLE
Added mutpy-pynguin as a package.

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge_admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge-admin, please restart ci

**Comment by thewchan** [comment]
@conda-forge/staged-recipes This is FINALLY ready to be reviewed, woohoo!

File: recipes/mutpy-pynguin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - mutpy ==0.6.1 pyhd8ed1ab_4
+    - pip
+    - python >=3.8
+  run:
+    - astmonkey >=0.3.6
+    - jinja2 >=2.7.1
+    - mutpy ==0.6.1 pyhd8ed1ab_4
```
Comment (dopplershift) [inline recipes/mutpy-pynguin/meta.yaml:26]: Upstream says this is a fork of `mutpy`...so why are you depending on it? I don't see it in the upstream requirements.

Comment (thewchan) [reply recipes/mutpy-pynguin/meta.yaml:26]: to be honest, it's been so long I don't remember. Let me try taking it ou.

Comment (thewchan) [reply recipes/mutpy-pynguin/meta.yaml]: looks like it's working? I really don't know/remember why I was trying to ad mutpy as a dependency. Thanks for catching this!

**Comment by thewchan** [comment]
@dopplershift thanks for checking! This is again good to go

**Review by dopplershift — APPROVED** [review]

**Pull request merged**